src/libraries/System.Security.Cryptography.Cose/tests/CoseSign1MessageTests.Verify.cs

```diff
@@ -116,6 +116,35 @@ public void VerifyReturnsFalseAfterAttemptWithCorrectContent()
             Assert.False(msg.Verify(DefaultKey, wrongContent), "Calling Verify with the wrong content");
         }
 
+        [Fact]
+        public void VerifyThrowsIfContentIsNull()
+        {
+            byte[] encodedMsg = CoseSign1Message.Sign(s_sampleContent, DefaultKey, DefaultHash);
+            CoseSign1Message msg = CoseMessage.DecodeSign1(encodedMsg);
+
+            Assert.Throws<ArgumentNullException>(() => msg.Verify(DefaultKey, null!));
```

Comment:
Should these be `AssertExtensions.Throws` (and throughout)?

Reply:
Ah, so we can check the Argument Name.